tajo-catalog/tajo-catalog-server/src/test/java/org/apache/tajo/catalog/TestCatalog.java

```diff
@@ -995,6 +995,10 @@ public void testAlterTableName () throws Exception {
     TableDesc addColumnDesc = catalog.getTableDesc("default","mynewcooltable");
     assertTrue(addColumnDesc.getSchema().containsByName("mynewcol"));
 
+    //SET_PROPERTY
+    catalog.alterTable(createMockAlterTableSetProperty());
+    TableDesc setPropertyDesc = catalog.getTableDesc("default","mynewcooltable");
+    assertEquals("GMT+0", setPropertyDesc.getMeta().getOption("timezone"));
```

Comment:
You already implemented an unit test case for this feature. But I suggest for you to implement more unit test cases because you just used mockup table. It would be more valuable if you verify the change status of TableDesc. Unit test cases for partitioned table would be help for you to implement this.